docker/tests/Dockerfile-fedora-test

```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: 2023 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+ARG dist
+FROM fedora:${dist}
+
+WORKDIR /tezos-packaging/docker
+
+RUN dnf update -y
+RUN dnf install -y python3-devel python3-setuptools
+RUN sudo dnf update -y
+RUN sudo dnf install -y 'dnf-command(copr)'
+RUN sudo dnf copr enable -y @Serokell/Tezos
```

Comment:
We should also check that packages from `Tezos-rc` repo get installed correctly, am I right? @pasqu4le 
```
ARG repo
RUN dnf copr enable -y @Serokell/${repo}
```

Reply:
seems like you forgot this one